app/Spago.hs

```diff
@@ -139,7 +149,7 @@ install = do
   config <- ensureConfig
   let deps = getAllDependencies config
   echo' $ "Installing " <> Text.pack (show $ List.length deps) <> " dependencies."
-  _ <- traverse getDep deps
+  _ <- forConcurrently_ deps getDep
```

Comment:
I think this is good, but Travis toasters can't handle this. Not very much worth looking into at the momen tthough.

Reply:
Yeah, and if need be this is quite easy to limit, e.g. see [how `psc-package` does this](https://github.com/purescript/psc-package/blob/648da70ae9b7ed48216ed03f930c1a6e8e902c0e/app/Main.hs#L269)

Reply:
Seems like something where http://hackage.haskell.org/package/async-pool-0.9.0.2/docs/Control-Concurrent-Async-Pool.html should be solving the problem